handleMeta.js

```diff
@@ -0,0 +1,15 @@
+import getViewport from "./getViewport"
+
+export function handleMeta (metaContent) {
```

Comment:
Чет у тебя тут перемудрено. Можно чуть попроще. Чтоб лишних переменных не собирать и коллекций при сборке нод
```suggestion
export function handleMeta (metaContent) {
  let meta = getViewport()

    if (!meta) {
      meta = document.createElement("meta")
      meta.name = "viewport"
      document.head.appendChild(meta)
    }

    meta.content = metaContent
 }
```

Можно еще декомпозировать, но и так сойдет, кмк